src/types.tsx

```diff
@@ -2,7 +2,7 @@ export interface IPost {
   pid?: number;
   uid?: number;
   title?: string;
-  content?: string;
+  content: string;
```

Comment:
Taking out the question mark allowed my filter to work, although it's still not working completely.